LayoutTests/resources/ui-helper.js

```diff
@@ -369,21 +369,20 @@ window.UIHelper = class UIHelper {
         });
     }
 
-    static activateAt(x, y, modifiers=[])
+    static async activateAt(x, y, modifiers=[])
     {
-        if (!this.isWebKit2() || !this.isIOSFamily()) {
-            eventSender.mouseMoveTo(x, y);
-            eventSender.mouseDown(0, modifiers);
-            eventSender.mouseUp(0, modifiers);
-            return Promise.resolve();
+        if (this.isWebKit2() && this.isIOSFamily()) {
+            return new Promise((resolve) => {
+                testRunner.runUIScript(`
+                    uiController.singleTapAtPointWithModifiers(${x}, ${y}, ${JSON.stringify(modifiers)}, function() {
+                        uiController.uiScriptComplete();
+                    });`, resolve);
+            });
         }
 
-        return new Promise((resolve) => {
-            testRunner.runUIScript(`
-                uiController.singleTapAtPointWithModifiers(${x}, ${y}, ${JSON.stringify(modifiers)}, function() {
-                    uiController.uiScriptComplete();
-                });`, resolve);
-        });
+        await eventSender.asyncMouseMoveTo(x, y);
+        await eventSender.asyncMouseDown(0, modifiers);
+        await eventSender.asyncMouseUp(0, modifiers);
```

Comment:
I don't think we should make this change in this PR. Many other tests use activateAt and this may change their behavior

Reply:
Yeah looks like it's causing failures. Fixed.